### ISSUE TITLE
Remove ‘Enter a longitude’ text in the meeting editor when auto-geocoding is enabled

### ISSUE THREAD

**Issue by rogersearle** [bmlt-enabled/bmlt-root-server#369]
There are descriptions under the Longitude and Latitude fields ("Enter a longitude") which is confusing, since the description on the right of the field says, is "Automatically calculated on save" and this data should NOT be entered manually?

**Comment by jbraswell** [comment]
I agree this is confusing. In the upcoming version, these field descriptions have been relocated to below the field. We could change it to something actually describing the field (for example, "The longitude of the meeting:), rather that a confusing message that seems to be directing the user to enter data into a read-only field.

Or... as suggested, we could remove the description completely. The label "Longitude" or "Latitude" is probably self-explanatory.
